templates/worker-turso-ts/README.md

```diff
@@ -0,0 +1,12 @@
+# CloudFlare worker using Turso and TypeScript
+
+This is a complete project that uses [Turso], an edge-hosted distributed
+database, and TypeScript to implement routes that read and write the database.
+It is provided as part of a [tutorial], which walks you through the process of
+setting up Turso and creating a database.
+
+In order to get this code to run, you will need to edit wrangler.toml and
+provide values for `LIBSQL_DB_URL` and `LIBSQL_DB_AUTH_TOKEN` that point to a database that's already been created according to the tutorial.
+
+[Turso]: https://chiselstrike.com/
+[tutorial]: https://developers.cloudflare.com/workers/tutorials/worker-turso-ts
```

Comment:
```suggestion
[Tutorial]: https://developers.cloudflare.com/workers/tutorials/connect-to-turso-database-from-workers/
```

Reply:
We intend to have another tutorial for rust.  Does that change the proposed URL for this one?

Reply:
I'd make the Rust one `/connect-to-turso-database-using-rust/` (since Workers == TS/JS to most users)

Reply:
Sounds good.